lib/rules/declaration-block-trailing-semicolon/README.md

```diff
@@ -88,3 +88,21 @@ a { color: pink }
 ```css
 a { background: orange; color: pink }
 ```
+
+## Optional secondary options
+
+### `ignore: ["single-declaration"]`
+
```

Comment:
How about add a brief description about this secondary option. For example:

```suggestion

Ignore the case when there is a single declaration within a block.
```

Reply:
Good catch. 

Let's go with "Ignore declaration blocks that contain a single declaration" to be [consistent with the form we usually use](https://github.com/stylelint/stylelint/blob/master/lib/rules/at-rule-empty-line-before/README.md#ignore-after-comment-first-nested-inside-block-blockless-after-same-name-blockless-blockless-after-blockless).